File: src/main/java/com/example/demo/article/dto/ArticleForm.java
package com.example.demo.article.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ArticleForm {
    public String title;
    public String content;
    public String pictureUrl;
    //private Map<AttachmentType, List<MultipartFile>> attachmentFiles = new ConcurrentHashMap<>();
}

File: src/main/kotlin/com/classy/placelog/article/dto/ArticleForm.kt
package com.classy.placelog.article.dto

import org.springframework.web.multipart.MultipartFile

/** 게시글 생성/수정 폼 */
data class ArticleForm(
    /** 제목 */
    val title: String,
    /** 내용 */
    val content: String,
    /** 이미지 파일 */
    val images: List<MultipartFile>? = null,
)
